screeps-game-api/javascript/utils.js

```diff
@@ -188,6 +190,7 @@ function __resource_type_str_to_num(str) {
         case RESOURCE_CATALYZED_ZYNTHIUM_ALKALIDE: return 41;
         case RESOURCE_CATALYZED_GHODIUM_ACID: return 42;
         case RESOURCE_CATALYZED_GHODIUM_ALKALIDE: return 43;
+        case RESOURCE_OPS: return 44;
```

Comment:
I think it has something to do with power creeps? There's a power named `PWR_GENERATE_OPS` which seems like it could be related.

I found the constant `RESOURCE_OPS` in the constants file at https://github.com/screeps/common/blob/068343753adf090fd1991944d2292be9e236b7dd/lib/constants.js#L406, and it is counted as part of "all resources" like the other ones at https://github.com/screeps/common/blob/068343753adf090fd1991944d2292be9e236b7dd/lib/constants.js#L989.

Reply:
It does seem to be used in engine: at https://github.com/screeps/engine/blob/a4e431c9ee33dcc55c04f3adc479145c210c73fd/src/processor/intents/power-creeps/usePower.js#L58.